src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java

```diff
@@ -18,6 +18,7 @@
 public class RecentWikiActivityTest extends NewTestTemplate {
 
   @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
+  @Execute(mockAds = "true")
```

Comment:
Good point, @SebastianMarzjan! :) We've got [tests for interstitial](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java#L42) the most important cases for us are tested it. However, we won't mind if somebody adds to it test checking if the close button works :japanese_ogre: